zebra-consensus/src/block/tests.rs

```diff
@@ -101,7 +105,10 @@ async fn check_transcripts_test() -> Result<(), Report> {
 #[spandoc::spandoc]
 async fn check_transcripts() -> Result<(), Report> {
     zebra_test::init();
-    let state_service = zebra_state::in_memory::init();
+
+    let network = Network::Mainnet;
+    let state_service = on_disk::init(Config::ephemeral(), network);
```

Comment:
Using `on_disk` and `ephemeral` in the same line is really confusing.

Let's re-export `init` from the top level of `zebra_state`?

```suggestion
    let state_service = zebra_state::init(Config::ephemeral(), network);
```